### ISSUE TITLE
If you know other organizations not listed here, please comment below

### ISSUE THREAD

**Issue by chhantyal** [chhantyal/PreyForNepal#1]
Please don't forget to give name and link to relief fund. Thank you.

**Comment by jackphelps** [comment]
CARE. They have an A rating with charitywatch.
http://www.care.org/
https://www.charitywatch.org/ratings-and-metrics/care/396

This list would be nicer if you included the chartywatch link for each org. Peoples' money goes way, way further when huge chunks of it aren't spend on overheard.

**Comment by Susannamp** [comment]
Educate the Children is a very good organization working in Nepal, and is also a US 501c3 nonprofit organization. 
http://www.etc-nepal.org/ 
http://www.etc-nepal.org/earthquake_relief.php
https://www.globalgiving.org/donate/3732/educate-the-children/

**Comment by chhantyal** [comment]
@jackphelps Thanks. Added care to the list. This list was made in hurry, Charitywatch looks nice. I am also looking into http://www.charitynavigator.org/

**Comment by ddgun** [comment]
MSF

http://www.msf.org/article/nepal-msf-sending-medical-and-non-medical-teams-assist-victims-nepal-earthquake

**Comment by sethia** [comment]
https://pmnrf.gov.in/ 
Prime Minister's National Relief Fund, (PMNRF) , 
Since India is helping Nepal as well as for relief in Indian states of Bihar/UP

**Comment by chhantyal** [comment]
Hey @sethia it's not clear whether the fund is going to be used for said disaster.

**Comment by sethia** [comment]
This should be a good proof, a lot of Indians (in tens thousands atleast) would be in Nepal and donation to PMNRF would definitely help Nepal directly/indirectly.
The best source I could find:  http://www.ndtv.com/india-news/after-nepal-earthquake-prime-minister-narendra-modi-donates-one-months-salary-to-relief-fund-758583

**Comment by jchaudhary** [comment]
German Red Cross - https://www.drk.de/spenden/jetzt-online-spenden.html

**Comment by ysinha** [comment]
Reddit is maintaining a list @ https://www.reddit.com/r/Nepal/comments/33wfxi/list_of_legitimate_charities_to_donate_to_nepal/

**Comment by deepsadhi** [comment]
Action Works Nepal
http://www.actionworksnepal.com/

**Comment by deepsadhi** [comment]
One Bucket of Relief Material for Earthquake Victims Nepal
http://bit.ly/quake_nepal

**Comment by deepsadhi** [comment]
http://eq.nitc.gov.np/
Money raised from Prime Minister Disaster Relief Fund is being used in the rescue, treatment, relief, rehabilitation of victims and restoration of physical infrastructure damaged by the earthquake.

**Comment by davidsth** [comment]
I agree with @deepsadhi Nepal has it's own PM relief fund which was specifically made for these kind of situations. Don't use the indian relief fund. 

[nepal embassy has information on the PM relief fund](http://www.nepalembassyusa.org/)

[here is a link that explains it's credibility](https://www.facebook.com/swarnim.wagle/posts/10102063234099121?pnref=story).

**Comment by neurotrophik** [comment]
Diyalo Foundation -- providing education to rural Nepali's who previously had zero access. 

www.diyalofoundation.org
